package.json

```diff
@@ -62,5 +62,8 @@
     ],
     "reporter": "html",
     "all": true
+  },
+  "volta": {
+    "node": "16.18.0"
```

Comment:
Previously we weren't pinning the Node version. Since this package isn't (or shouldn't be) likely to break with Node updates, I would be inclined to run tests against the current Node LTS release.

Reply:
Oh! I thought v16 was the LTS, whoops! My motivation for adding this is that I am working in multiple repos that require various versions of Node, some of which I can't control how they specify which Node version to use. So my goal for adding this was to make sure I'm using the latest and greatest when developing in this repo (I meant to put this at the LTS).

If you'd prefer not to pin this repo for users of volta or for the workflow, I'm happy to remove.